src/select/internal.tsx

```diff
@@ -155,6 +155,7 @@ const InternalSelect = React.forwardRef(
         {...formFieldContext}
         controlId={controlId}
         ariaLabelledby={joinStrings(formFieldContext.ariaLabelledby, selectAriaLabelId)}
+        ariaLabel={ariaLabel}
```

Comment:
No need to add ariaLabel in Select because Select does it itself already. You can remove this line and the change in select/parts. https://github.com/cloudscape-design/components/blob/ab8601ce532c2dd60a563629c261d7da2d8f8df8/src/select/internal.tsx#L233

Reply:
@YueyingLu this is fixed now.